packages/backend/src/core/DiscoveryWatcher.ts

```diff
@@ -209,11 +215,20 @@ export class DiscoveryWatcher {
       return
     }
 
-    for (const message of messages) {
-      await this.discordClient.sendMessage(message, channel).then(
-        () => this.logger.info('Notification to Discord has been sent'),
-        (e) => this.logger.error(e),
-      )
+    const arrayMessages = Array.isArray(messages) ? messages : [messages]
+    for (const message of arrayMessages) {
+      type Channel = 'PUBLIC' | 'INTERNAL'
```

Comment:
That's the first time i see a type definition inside the function. You sure you want to leave it here?